applications/sia/secrets.yaml

```diff
@@ -0,0 +1,20 @@
+"aws-credentials.ini":
+  copy:
+    application: nublado
+    key: "aws-credentials.ini"
+"butler-gcs-idf-creds.json":
+  copy:
+    application: nublado
+    key: "butler-gcs-idf-creds.json"
+"postgres-credentials.txt":
+  copy:
+    application: nublado
+    key: "postgres-credentials.txt"
```

Comment:
It would be better if we could make these all conditional on enabling support for local Butler. I think that can't be done right now given the layout of `values.yaml` because this is a property of each individual repository, so maybe we could disable local Butler support by default and add a switch to enable it just if we wanted to test it?

I want to make sure we don't give the SIA service those secrets during normal operation, since we want to make sure we're only using remote Butler.